From: Richard Cochran <richardcochran@gmail.com>
Subject: net: export the receive time stamping hook for non-NAPI drivers
Git-commit: 238442f6bc10ba31577fc47e476c88689c80c7fa
Patch-mainline: 3.1
References: fate#314848
Acked-by: Jiri Bohac <jbohac@suse.cz>

Ethernet MAC drivers based on phylib (but not using NAPI) can
enable hardware time stamping in phy devices by calling netif_rx()
conditionally based on a call to skb_defer_rx_timestamp().

This commit exports that function so that drivers calling it may
be compiled as modules.

Signed-off-by: Richard Cochran <richard.cochran@omicron.at>
Signed-off-by: David S. Miller <davem@davemloft.net>

diff --git a/net/core/timestamping.c b/net/core/timestamping.c
index 97d036a..c799b0a 100644
--- a/net/core/timestamping.c
+++ b/net/core/timestamping.c
@@ -129,6 +129,7 @@ bool skb_defer_rx_timestamp(struct sk_buff *skb)
 
 	return false;
 }
+EXPORT_SYMBOL_GPL(skb_defer_rx_timestamp);
 
 void __init skb_timestamping_init(void)
 {
-- 
1.7.11.4

